packages/constants/src/payload/payload.types.ts

```diff
@@ -24,10 +24,21 @@ export interface PaymentRequestPayload {
   currency?: string;
   // The issuer of the token
   issuer?: string;
-  // The memo to attach to the transaction
-  memo?: string;
+  // The memos to attach to the transaction
+  // Each attribute of each memo must be hex encoded.
+  memos?: Memo[];
   // The destination tag to attach to the transaction
-  destinationTag?: string;
+  destinationTag?: number;
+  // Integer amount of XRP, in drops, to be destroyed as a cost for distributing this transaction to the network.
+  fee?: string;
+}
+
+export interface Memo {
```

Comment:
Is there a way to import this type from XRPL.JS?

Reply:
I did first the it would create inconsistencies in the naming convention (CamelCase with the first letter capitalized versus camelCase with the first letter lowercased)
Have a look at my comment here: https://github.com/GemWallet/gemwallet-extension/pull/152#discussion_r1184160502